pkg/otelcollector/consts/consts.go

```diff
@@ -61,10 +61,10 @@ const (
 	ApertureLoadSchedulersLabel = "aperture.load_schedulers"
 	// ApertureDroppingLoadSchedulersLabel describes load schedulers dropping the traffic.
 	ApertureDroppingLoadSchedulersLabel = "aperture.dropping_load_schedulers"
-	// ApertureLoadRegulatorsLabel describes load regulators matched to the traffic.
-	ApertureLoadRegulatorsLabel = "aperture.load_regulators"
-	// ApertureDroppingLoadRegulatorsLabel describes load regulators dropping the traffic.
-	ApertureDroppingLoadRegulatorsLabel = "aperture.dropping_load_regulators"
+	// ApertureLoadSamplersLabel describes load samplers matched to the traffic.
+	ApertureLoadSamplersLabel = "aperture.load_samplers"
```

Comment:
just call it samplers?

Reply:
Just sampler sounds odd.

Reply:
Making the change afterall.

Reply:
[649485d](https://github.com/fluxninja/aperture/pull/2140/commits/649485d0bc55002b87cbd75b3f43e61b7f4efcff) addresses this.